dts/bindings/pinctrl/microchip,xec-pinctrl-conf.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 Linaro Limited
+# Copyright (c) 2021 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Microchip XEC Pin controller Node
+    Based on pincfg-node.yaml binding.
+
+    Note: `bias-disable` and `drive-push-pull` are default pin configurations.
+           They will be applied in case no `bias-foo` or `driver-bar` properties
+           are set.
```

Comment:
please try to extend this documentation as other bindings do, refer to nrf/gd32/ etc.

Reply:
okay

Reply:
updated documentation